### ISSUE TITLE
Add page titles to pages within the app

### ISSUE THREAD

**Issue by robby1066** [robby1066/keepposted-help#24]
# What's the problem you're hoping this new feature will solve? 
Page titles are a useful navigational aid. Titles identify the current location without requiring users to read or interpret page content. User agents make the title of the page easily available to the user for identifying the page. For instance, a user agent may display the page title in the window title bar or as the name of the tab containing the page.

# Description of feature
Each page within the Keep Posted app should have a descriptive page title defined in the `TITLE` tag

# Additional context
https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html

**Comment by robby1066** [comment]
Deployed on Feb 5, 2021 - Every page within the app should now have a descriptive page title.

**Comment by robby1066** [comment]
Example page titles

![image](https://user-images.githubusercontent.com/342510/107127393-7188f800-687b-11eb-9f93-4c055bf09c87.png)
